js/slider/Index.svelte

```diff
@@ -24,7 +24,7 @@
 	export let scale: number | null = null;
 	export let min_width: number | undefined = undefined;
 	export let minimum: number;
-	export let maximum: number;
+	export let maximum = 100;
```

Comment:
Default for browser is 100 anyway, so just set this explicitly, to avoid undefined values.